eclipse-scout-core/test/dataobject/dataObjectsSpec.ts

```diff
@@ -23,10 +23,10 @@ describe('dataObjects', () => {
 
   describe('addContribution', () => {
     it('adds a contribution to the DO', () => {
-      let doEntity: DoEntityWithContributions = {};
+      const doEntity: DoEntityWithContributions = {};
```

Comment:
we don't have a guideline for let vs const (yet). No need to replace let with const everywhere. I personally prefer let over const